volume/local/local.go

```diff
@@ -22,6 +22,7 @@ const (
 
 var oldVfsDir = filepath.Join("vfs", "dir")
 
+// New looks at each directory in the given scope and adds it to the returned Root.
```

Comment:
New instantiates a new Root instance with the provided scope. Scope is the base path to that the Root instance uses to store its volumes. The base path is created here if it does not exist.